generated/nidcpower/unit_tests/mock_helper.py

```diff
@@ -231,21 +231,37 @@ def niDCPower_ExportSignal(self, vi, signal, signal_identifier, output_terminal)
     def niDCPower_FetchMultiple(self, vi, channel_name, timeout, count, voltage_measurements, current_measurements, in_compliance, actual_count):  # noqa: N802
         if self._defaults['FetchMultiple']['return'] != 0:
             return self._defaults['FetchMultiple']['return']
+        # voltage_measurements
         if self._defaults['FetchMultiple']['voltageMeasurements'] is None:
             raise MockFunctionCallError("niDCPower_FetchMultiple", param='voltageMeasurements')
         a = self._defaults['FetchMultiple']['voltageMeasurements']
```

Comment:
Looking at this code made me wish variable `a` had a more descriptive name.

Reply:
Renamed to `test_value`